doc/classes/TabContainer.xml

```diff
@@ -151,7 +156,7 @@
 			<argument index="1" name="title" type="String">
 			</argument>
 			<description>
-				Set a title for the tab at index [code]tab_idx[/code]. Tab titles are by default the children node name, but this can be overridden.
+				Set a title for the tab at index [code]tab_idx[/code]. Tab titles default to the name of the indexed child node, but this can be overridden with [method set_tab_title].
```

Comment:
Set -> Sets

Sets the title of the tab...

Reply:
Fixed